match.c

```diff
@@ -426,9 +429,12 @@ ZLIB_INTERNAL unsigned longest_match(deflate_state *const s, IPos cur_match) {
         scan += 2, match+=2;
         Assert(*scan == *match, "match[2]?");
         do {
-            unsigned long sv = *(unsigned long*)(void*)scan;
-            unsigned long mv = *(unsigned long*)(void*)match;
-            unsigned long xor = sv ^ mv;
+            unsigned long sv, mv. xor;
```

Comment:
Should be "unsigned long sv, mv, xor;"
There is period instead of comma between "mv" and "xor".

Reply:
How did this slip through? how did it even compile :(